src/unit_tests/wrappers/externals/sqlite/sqlite3_wrappers.c

```diff
@@ -196,3 +200,7 @@ const char* __wrap_sqlite3_column_name(__attribute__((unused)) sqlite3_stmt *pSt
 int __wrap_sqlite3_changes(__attribute__((unused)) sqlite3 * db){
     return mock();
 }
+
+int __wrap_sqlite3_get_autocommit(__attribute__((unused)) sqlite3 * db) {
+    return mock();
+}
```

Comment:
Please add a new line at the end of this file

Reply:
Fixed!